asset/src/main/java/com/example/asset/CreateSavedQueryExample.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.asset;
+
+// [START asset_quickstart_create_saved_query]
+import com.google.cloud.asset.v1.AssetServiceClient;
+import com.google.cloud.asset.v1.CreateSavedQueryRequest;
+import com.google.cloud.asset.v1.IamPolicyAnalysisQuery;
+import com.google.cloud.asset.v1.IamPolicyAnalysisQuery.AccessSelector;
+import com.google.cloud.asset.v1.ProjectName;
+import com.google.cloud.asset.v1.SavedQuery;
+import java.io.IOException;
+import java.util.Arrays;
+
+public class CreateSavedQueryExample {
```

Comment:
Please add a main class or an createSavedQuery() to call the sample, see https://github.com/GoogleCloudPlatform/java-docs-samples/blob/main/SAMPLE_FORMAT.md#method-structure

Reply:
Done.